db/migrate/20230510112953_add_service_name_to_active_storage_blobs.active_storage.rb

```diff
@@ -0,0 +1,22 @@
+# This migration comes from active_storage (originally 20190112182829)
```

Comment:
Why add the active storage tables? (other than "so we can use active storage in the future if we want and the tables are all there ready to go")